portal/src/routes/(authenticated)/organizations/[id]/settings/products/+page.server.ts

```diff
@@ -39,7 +39,8 @@ export const actions = {
   async togglePublic(event) {
     const form = await superValidate(event.request, valibot(togglePublicSchema));
     if (!form.valid) return fail(400, { form, ok: false });
-    if (form.data.orgId !== parseInt(event.params.id)) return fail(404, { form, ok: false });
+    // if user modified hidden values
+    if (form.data.orgId !== parseInt(event.params.id)) return fail(403, { form, ok: false });
```

Comment:
Here we should just use `parseInt(event.params.id)` and leave `orgId` out of the edit schema.